src/main/java/org/apache/commons/lang3/builder/ReflectionToStringBuilder.java

```diff
@@ -385,6 +385,19 @@ public static String toStringExclude(final Object object, final Collection<Strin
         return toStringExclude(object, toNoNullStringArray(excludeFieldNames));
     }
 
+    /**
+     * Builds a String for a toString method with only the selected field names.
+     *
+     * @param object
+     *            The object to "toString".
+     * @param selectedFieldNames
+     *            The field names that must be on toString. Null selects nothing.
```

Comment:
in case of Null parameter, What is the output:

Reply:
I see. Can you add these details in the JavaDocs? When reading I got the impression that Null would return an empty string.